.travis.yml

```diff
@@ -2,7 +2,7 @@ language: scala
 
 # Only build non-pushes (so PRs, API requests & cron jobs) OR tags OR forks
 # https://docs.travis-ci.com/user/conditional-builds-stages-jobs/
-if: type != push OR tag IS present OR repo != lagom/lagom
+if: tag IS present OR type != push OR (branch IN (master, 1.5.x, 1.4.x) AND type = push)
```

Comment:
I'm not sure about these rules actually. I didn't find any clear documentation about what a `push` means. 

Considering that the available options for `type` are: `push`, `pull request`, `api` and `cron`. I'm assuming that `push` means any direct commit on a branch. 

So, with `type != push` we want to build any PR, API calls and cron jobs. 

With `(branch IN (master, 1.5.x, 1.4.x) AND type = push)`, I'm trying to rule that if there is a push, we only build if it's on one those branches. The goal here is to avoid double builds when we open a PR from a branch in `lagom/lagom`. 

Not sure if that will work tough.

Reply:
`repo != lagom/lagom OR type != push OR branch IN (master, 1.5.x, 1.4.x)`

Reply:
`type != push OR tag IS present OR repo != lagom/lagom OR branch IN (master, 1.5.x, 1.4.x)`

Reply:
`type != push OR tag IS present OR repo != lagom/lagom OR (branch IN (master, 1.5.x, 1.4.x) AND type = push)`

Reply:
`type != push OR tag IS present OR repo != lagom/lagom OR branch IN (master, 1.5.x, 1.4.x)`